### DISCUSSION TITLE
Pull method in Laravel Collections

### DISCUSSION THREAD

**Discussion by glupeksha** [laravel/framework#35064] [General] [answered]
In laravel collections, pull method require an key to pull an item from the collection. Shouldnt it pull the last item if a key is not specified?

**Comment by selcukcukur** [comment, marked as answer]
@upeksha1996 This situation can cause a lot of unusual situations. If no key is specified, it is better to return blank.

**Reply by glupeksha** [reply]
We can push without specifying a key, but why not pull? I think it makes more sense to pull the last item than specifying a key. And the name will be consistent with other data structures like stacks and queues.

**Reply by rojtjo** [reply]
Isn't that what `->pop()` is for? Like `array_pop()`

**Reply by glupeksha** [reply]
How come I missed it :3 @rojtjo thaanks
